### PR TITLE
Frigide Recipe Balance

### PR DESCRIPTION
**Pull request by DaChickenGamer**
Made it so it takes EOW and BOC drops so you can't speedrun hell stone before bosses. Also some minor changes like the git ignore and moving the BootRecipes.

### CODE REVIEW

File: Content/RecipeChanges/FrigidRecipeChange.cs
Comment (1Git2Clone) [inline Content/RecipeChanges/FrigidRecipeChange.cs:1]: You can rename this file to `Content/RecipeChanges/FrigidPickaxe.cs` it's obvious it's a recipe change from the parent directory.

File: Content/RecipeChanges/FrigidRecipeChange.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+                Recipe recipe = Main.recipe[i];
+
+                if (recipe.createItem.type != ModContent.ItemType<FrigidPickaxe>()) continue;
+                
+                Recipe alternativeRecipe = recipe.Clone();
+                
+                recipe.AddIngredient(ItemID.TissueSample, 6);
+
+                alternativeRecipe.AddIngredient(ItemID.ShadowScale, 6);
+                alternativeRecipe.Register();
```
Comment (1Git2Clone) [inline Content/RecipeChanges/FrigidRecipeChange.cs:24]: Can't we just use the original `recipe` variable here?

File: .gitignore
Comment (1Git2Clone) [inline .gitignore:1]: Looks good to me!

File: Content/RecipeChanges/BootRecipes.cs
Comment (1Git2Clone) [inline Content/RecipeChanges/BootRecipes.cs:1]: Looks good to me! Could also refactor it to a file per boot really, I'll think about that on a different PR though so don't worry.

File: TheBereftSouls.csproj
Comment (1Git2Clone) [inline TheBereftSouls.csproj:1]: Looks good to me! Interesting that this has to be included though, but alright.

**Review by 1Git2Clone — COMMENTED** [review]
I can't test the code in order to accept the changes but it's mostly good from what I see, you can look at the comments for: `Content/RecipeChanges/FrigidRecipeChange.cs` to see what can be changed.

File: Content/RecipeChanges/FrigidRecipeChange.cs (hunk 1 above)
Comment (DaChickenGamer) [reply Content/RecipeChanges/FrigidRecipeChange.cs:24]: I was making two different recipes which required two different variables I believe or else it'd give the recipe both items.

**Pull request merged**